source/ptreferential/ptreferential_ng.cpp

```diff
@@ -294,22 +294,23 @@ struct Eval : boost::static_visitor<Indexes> {
         } else if (f.type == "disruption"
                    && ((f.method == "tag" && f.args.size() == 1) || (f.method == "tags" && f.args.size() >= 1))) {
             indexes = get_impacts_by_tags(f.args, data);
-        } else if ((f.method == "since" && f.args.size() == 2) || (f.method == "until" && f.args.size() == 2)
-                   || (f.method == "between" && f.args.size() == 3)) {
+        } else if ((f.method == "since" && f.args.size() == 1 + (f.type == "vehicle_journey"))
```

Comment:
So, now `since, until and between` are only available for vehicle_journey type ?

Reply:
No, they are still available for VJ and Disruption (for other PTobjects its accepted here but rejected after, inside `filter_on_periodfilter_on_period` like it was the case before).

Here the point is for VJ we check that there is one more arg (the `data_freshness`), while there is not for others.

But it may not be super clear from the code, I'm open to suggestion if you think it should be rewritten/commented.

Reply:
okay I didn't understand because of the `+`. In my reading, I interpreted that like a `&&` (like a _reflex_)